### PR TITLE
Add support for ping events to infosquito

### PR DESCRIPTION
**Pull request by johnworth**
This pull request contains the changes needed to allow infosquito to respond to event messages.

### CODE REVIEW

File: src/infosquito/messages.clj [hunk 1]
```diff
@@ -62,6 +64,28 @@
...
+
+(def handlers
+  {"index.all"              reindex-handler
+   "index.data"             reindex-handler
+   "events.infosquito.ping" ping-handler})
+
+(defn- message-router
+  [props channel {:keys [routing-key] :as metadata} msg]
+  (let [handler (get handlers routing-key)]
+    (if-not (nil? handler)
```
Comment (slr71) [inline src/infosquito/messages.clj:86]: It might be helpful to use `when-not` here instead.

**Review by slr71 — APPROVED** [review]
I had one extremely minor suggestion, but the changes look good to me. :shipit: whenever you're ready.

File: src/infosquito/messages.clj [hunk 2]
```diff
@@ -48,7 +50,7 @@
               {:durable     true
                :auto-delete false
                :exclusive   false})
-  (doseq [key ["index.all" "index.data"]]
+  (doseq [key ["index.all" "index.data" "events.infosqito.#"]]
```
Comment (ianmcorvidae) [inline src/infosquito/messages.clj:53]: typo in this line, might cause issues

Comment (slr71) [reply src/infosquito/messages.clj:53]: Heh, I completely missed that. Good catch.

Comment (johnworth) [reply src/infosquito/messages.clj:53]: Should be fixed now, thanks.

**Pull request merged**